### DISCUSSION TITLE
Please get Firefox's AMO Recommended badge (msg for forks as well)

### DISCUSSION THREAD

**Discussion by OutshineIssue** [einaregilsson/Redirector#410] [General]
Redirector has quite some unrestricted permissions...

![image](https://github.com/user-attachments/assets/6d474aa7-1b46-410e-b748-4e0114fe0bee)

While it may seem like I'm making a mountain out of a molehill by bringing this up regarding an open-source add-on on its own GitHub repository, I should note that I'm not a developer, and neither is the majority of the user base. So, even though the code is publicly available, it still is a "trust me, bro" situation for most people.

I know these permissions are crucial for the add-on to work, I'm not discussing that, I just want to suggest taking a look at the [Recommended Extensions program](https://support.mozilla.org/en-US/kb/recommended-extensions-program) made by Mozilla where they review and give a trusted badge so there's no warning when installing:

![image](https://github.com/user-attachments/assets/07b0d534-9195-4fc9-9c9e-c8b0f24201ce)

I'm aware of what happened to the original creator and that this repository is essentially in maintenance mode under @Gitoffthelawn. So I'm not sure how likely it is that we'll see a change on the AMO page, but this message is also intended for all the forks out there.

**Comment by Gitoffthelawn** [comment]
Good idea.

Did Einar have a Recommended badge at one point?  I thought so, but I may be misremembering.

**Comment by OutshineIssue** [comment]
I'm not sure, jugding by userbase seems like it should if proposed. I've noticed badges on extensions with less than 8k users, while Redirector is getting close to 17k. Code size shouldn't be a problem either compared to other badge-approved add-ons, 51.72 KB seems like a short review.

**Reply by Gitoffthelawn** [reply]
I wonder if Mozilla is still approving applications for Recommendation badges.  They seemed to arbitrarily cap it at 100 at one point.  I do think they lost/removed a few, so are technically a little below that number now, though.

**Reply by OutshineIssue** [reply]
Yes, they are currently accepting submissions, but the extension must be popular, and I believe Redirector definitely qualifies. As I mentioned, 7k seems to be the floor.

**Reply by Gitoffthelawn** [reply]
> Yes, they are currently accepting submissions, but the extension must be popular, and I believe Redirector definitely qualifies. As I mentioned, 7k seems to be the floor.

Thanks! BTW, how did you find out they are currently accepting applications?
